bindings/electron/src/index.d.ts

```diff
@@ -230,8 +239,8 @@ export interface UserInfo {
 
 export interface WorkspaceInfo {
     id: string
-    name: string
-    selfCurrentRole: RealmRole
+    currentName: string
```

Comment:
The `current` prefix seems redundant: even if the name changes, it will always be the _current_ name isn't it?